src/Raven.Server/Documents/TcpHandlers/SubscriptionConnection.cs

```diff
@@ -142,24 +166,24 @@ public async Task InitAsync()
                 }
                 catch (TimeoutException)
                 {
-                    if (timeout == TimeSpan.Zero && _logger.IsInfoEnabled)
+                    if (timeout == InitialConnectionTimeout && _logger.IsInfoEnabled)
                     {
                         _logger.Info(
                             $"Subscription Id {SubscriptionId} from IP {TcpConnection.TcpClient.Client.RemoteEndPoint} starts to wait until previous connection from {_connectionState.Connection?.TcpConnection.TcpClient.Client.RemoteEndPoint} is released");
                     }
                     timeout = TimeSpan.FromMilliseconds(Math.Max(250, (long)_options.TimeToWaitBeforeConnectionRetry.TotalMilliseconds / 2));
-                    await SendHeartBeat();
+                    await SendHeartBeat($"Client from IP Waiting {TcpConnection.TcpClient.Client.RemoteEndPoint} waiting for subscription that is serving IP {_connectionState.Connection?.TcpConnection.TcpClient.Client.RemoteEndPoint} to be released");
```

Comment:
is this first `Waiting` needed?

Reply:
Yes, if the client died we want to know it as soon as possible

Reply:
@maximburyak I'm talking about the text here.

It will be: `Client from IP Waiting '127.0.0.1' waiting for...`

Reply:
It should be probably `Client from IP '127.0.0.1' waiting for...`

Reply:
Correct, too much copy pasting, will be fixed in next PR

Reply:
Already fixed: https://github.com/ravendb/ravendb/commit/912c34ef475d94d8c11a191024ea021a105af0c1